awaitwhat/graph.py

```diff
@@ -19,6 +19,19 @@ def new(tasks) -> Tuple[Set[Vertex], List[Edge]]:
     for who in tasks:
         src = Vertex.new(who, current)
         vertices.add(src)
+        children = blockers(who)
+        if len(children) > 1 and isinstance(children[1], asyncio.Task):
```

Comment:
I'm not happy about this, but the truth is, there had been an idea to change the data shape before. 👀